setup.py

```diff
@@ -46,7 +46,7 @@
         "gbd_mapping>=4.0.0",
         "layered_config_tree",
         "vivarium>=3.4.7",
-        "vivarium_public_health>=4.2.5",
+        "vivarium_public_health>=4.2.5,<4.3.0",
```

Comment:
I assume this means it was more complicated than just importing from the correct location?

Reply:
Im not sure why we can't just use the new path instead of pinning below this version